CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/Conversation.java

```diff
@@ -174,6 +181,17 @@ public GraphUpdateManager getGraphUpdateManager() {
         return graphUpdateManager;
     }
 
+    /**
+     * Inspects the contents of the conversation state and registers any changes
+     * that have occurred.
+     */
+    private UpdateComponent<GraphReadMethods> conversationExistanceUpdater = new UpdateComponent<GraphReadMethods>("Existance State", LOCK_STAGE) {
+
+        @Override
+        protected boolean update(GraphReadMethods graph) {
+            return Objects.isNull(graph);
```

Comment:
Is there any reason you have used `Objects.isNull`?

This is a minor feedback but to me `Objects.isNull` looks a bit out of place given the rest of the code base doesn't use this approach to check for `null`. Are you able to change this to `graph == null` so it looks more familiar? There are several occurrences of this method that would require changing in this PR.

Apparently it is useful inside lambda functions, see https://stackoverflow.com/a/46025437.

Reply:
No meaningful reason, just personal preference. I find it easier to read,

Changed.